typescript/src/languages/en-US/commands/giveaway.json

```diff
@@ -101,5 +101,36 @@
 			"30m 6h A hug from Skyra."
 		]
 	},
-	"giveawayRerollInvalid": "The message ID does not exist or there is no finished giveaway."
-}
\ No newline at end of file
+	"giveawayRerollInvalid": "The message ID does not exist or there is no finished giveaway.",
+	"giveawayEndDescription": "Ends a running giveaway message.",
```

Comment:
Format for Giveaway End could be :point_down:
```
:tada: Congrats <winner>! You won <Prize> 
<message_link> 
```
(Message link of the ended giveaway in next line.)

Reply:
No, it ends the giveaway, which makes it post this response, we don't need to duplicate this code.